modules/standard/IO.chpl

```diff
@@ -1973,27 +1973,12 @@ proc file.fsync() throws {
   if err then try ioerror(err, "in file.fsync", this._tryGetPath());
 }
 
-/*
-  A compile-time parameter to control the behavior of :proc:`file.path`
-
-  When 'false', the deprecated behavior is used (i.e., return the shortest of
-  the relative path and the absolute path)
-
-  When 'true', the new behavior is used (i.e., always return the absolute path)
-*/
-config param filePathAbsolute = false;
```

Comment:
It's probably nicer to users to make the config meaningless and generate a deprecation warning for it.  I would hope that most users attempting to use `file.path` had been relying on this config, so they'll get a compilation failure otherwise.